### PR TITLE
Handbook page: merge appendix C (FAQ) into a single file #148

### LINKED ISSUE #148: Handbook page: merge appendix C (FAQ) into a single file
Currently each FAQ is in a separate file. 
See appendix B (policies) for an example.

### PR DESCRIPTION
**Pull request by thenaesh**
Fixes #148
Here is a [preview](https://thenaesh.github.io/website/).

### CODE REVIEW

**Comment by damithc** [comment]
@thenaesh your preview version doesn't seem to work in the schedule page.
image

**Comment by thenaesh** [comment]
Ah ok, I identified the problem. I deleted all the original files and ended up leaving broken links. I'm working on a fix now.

**Comment by thenaesh** [comment]
I've fixed all the broken links. I've tested the schedule page and it works now. I searched through the other pages and didn't see any broken links, so no changes there.

File: contents/handbook-appendixC-faq.html [hunk 1]
```diff
@@ -1 +1,189 @@
...
+</div>
+
+<h3>Why do I have to come to tutorials on time?</h3>
+<div id="handbook-faq-timelyArrival">
+    <p>Unlike most modules, our tutorials are groupwork. Arriving on time minimizes the time wasted by 
+    others waiting for you to arrive or briefing you on what you missed. Being punctual is expected 
+    from a professional.</p>
+</div>
+
+<h3>What if I don’t carry around a laptop?</h3>
```
Comment (acjh) [inline contents/handbook-appendixC-faq.html]: Change all curly quotes (`’`) to straight quotes (`'`).

File: contents/handbook-appendixC-faq.html [hunk 2]
```diff
@@ -1 +1,189 @@
...
+    environment.</li>
+    </ul>
+    <p>Practical reasons:</p>
+    <ul>
+    <li>Some of the learning activities are tightly coupled to tools. Allowing more tools means 
+    tutors need to learn more tools, which increases their workload.</li>
+    <li>We provide learning resources for tools. e.g. 'Getting started with Eclipse'. 
+    Allowing more tools means we need to produce more resources.</li>
+    <li>When all students use the same tool, the collective expertise of the tool is more, 
+    increasing the opportunities for you to learn from each others.</li>
```
Comment (acjh) [inline contents/handbook-appendixC-faq.html]: Indent list items.

File: contents/handbook-appendixC-faq.html [hunk 3]
```diff
@@ -1 +1,189 @@
...
+    to eventually (after this module) migrate to better tools and, having learned more than one 
+    tool, to attain a more general understanding about a family of tools.</li>
+    <li>Learn to thrive under given conditions: As a professional Software Engineer, you must learn 
+    to be productive in any given tool environment, rather than insist on using your preferred 
+    tools. It is usually in small companies doing less important work that you get to chose your own 
+    toolset. Bigger companies working on mature products often impose some choices on developers, 
+    such as the project management tool, code repository, IDE, language etc. For example, Google 
+    uses SVN as their revision control software. Sometimes this is due to cost reasons (tool 
+    licensing cost), and sometimes due to legacy reasons (because the tool is already entrenched in 
+    their code base). <br />
```
Comment (acjh) [inline contents/handbook-appendixC-faq.html]: Remove whitespace between `.` and `<br />` (which gets parsed as an unnecessary `&nbsp;`).

File: contents/handbook-appendixC-faq.html [hunk 4]
```diff
@@ -1 +1,189 @@
...
+<h3>Why submission requirements differ between CS2103T and CS2101?</h3>
+<div id="handbook-faq-cs2101Differences">
+    <p><img src="../images/Submission requirements differ.png" width="300"  /></p>
+    <p>They do, and they should.</p>
+    <p>CS2103T communication requirements are limited to a very narrow scope (i.e., communicate about 
+    the product to users and developers). CS2101 aims to teach you technical communication in a much 
+    wider context. While you may be able to reuse some of the stuff across the two modules, 
+    submissions are not intended to be exactly the same.</p>
+</div>
+
```
Comment (acjh) [inline contents/handbook-appendixC-faq.html]: Remove additional newline at the end (one is good; there are actually two here).

File: contents/handbook-appendixC-faq.html [hunk 5]
```diff
@@ -1 +1,189 @@
...
+    to be productive in any given tool environment, rather than insist on using your preferred 
+    tools. It is usually in small companies doing less important work that you get to chose your own 
+    toolset. Bigger companies working on mature products often impose some choices on developers, 
+    such as the project management tool, code repository, IDE, language etc. For example, Google 
+    uses SVN as their revision control software. Sometimes this is due to cost reasons (tool 
+    licensing cost), and sometimes due to legacy reasons (because the tool is already entrenched in 
+    their code base). <br />
+    While programming in school is often a solo sport, programming in the industry is a team sport. 
+    As we are training you to become professional software engineers, it is important to get over 
+    the psychological hurdle of needing to satisfy individual preferences and get used to making the best of a given
```
Comment (acjh) [inline contents/handbook-appendixC-faq.html]: Move `best of a given` to the next line (on the same line as `environment`) to be consistent with line lengths.

File: contents/handbook-appendixC-faq.html [hunk 6]
```diff
@@ -1 +1,189 @@
...
+    <p>Intermediate submissions are also incremental submissions. Intermediate submissions are 
+    included in the final submission too and will earn marks at that point. They are there to help you 
+    do well in the final submission and for us to monitor your progress and guide you accordingly. 
+    They are a means to an end. They are not achievements themselves. By right, they shouldn't 
+    be given marks at all. :-)</p>
+</div>
+
+<h3>Why submission requirements differ between CS2103T and CS2101?</h3>
+<div id="handbook-faq-cs2101Differences">
+    <p><img src="../images/Submission requirements differ.png" width="300"  /></p>
```
Comment (acjh) [inline contents/handbook-appendixC-faq.html]: Remove additional whitespace between `"300"` and `/>` (one is good; there are actually two here).

File: contents/handbook-appendixC-faq.html [hunk 7]
```diff
@@ -1 +1,189 @@
...
+    extra time/effort from the tutor or team members.</p>
+    <p>Reason: You enjoy the benefits of not bring the laptop; you should bear the cost too</p>
+</div>
+
+<h3>Why very narrow project scope?</h3>
+<div id="handbook-faq-narrowScope">
+    <p>Defining your own unique project is more fun.</p>
+    <p>But, wider scope → more diverse projects → harder for us to go deep into your project. The 
+    collective know-how we (i.e., students and the teaching team) build up about SE issues related to 
+    the project become shallow and stretched too thinly.  It also affects fairness of grading.</p>
```
Comment (acjh) [inline contents/handbook-appendixC-faq.html]: Remove additional whitespace between `thinly.` and `It`.

File: contents/handbook-appendixC-faq.html [hunk 8]
```diff
@@ -1 +1,189 @@
...
+
+<h3>Why slides are not detailed enough, handouts are wordy?</h3>
+<div id="handbook-faq-slideFormat">
+    <p><img src="../images/slides not detailed enough.png" width="733" height="221" /></p>
+    <p>Slides are not meant to be documents to print and study for exams. Their purpose is to support 
+    the lecture delivery and keep you engaged during the lecture. That's why our slides are less 
+    detailed and more visual.</p>
+    <p>Handouts are wordy because they are reference materials for you  to study at your own pace and 
+    use during the module and later. The format is similar to materials (books, online articles, etc.) 
+    you will have to refer to  when you work in the industry. They too are not meant to be cheat
```
Comment (acjh) [inline contents/handbook-appendixC-faq.html]: Remove additional whitespace between:
- `reference materials for you` and `to study`,
- `refer to` and `when you work`.

**Comment by thenaesh** [comment]
@acjh I've made the changes requested.

**Comment by acjh** [comment]
Looks good. @damithc Any comments?

**Pull request merged**